frontend/src/scenes/notebooks/Notebook/notebookLogic.ts

```diff
@@ -518,6 +528,11 @@ export const notebookLogic = kea<notebookLogicType>([
             )
         },
         setLocalContent: async ({ updateEditor, jsonContent }, breakpoint) => {
+            if (values.notebook?.user_access_level !== 'editor') {
```

Comment:
Edge case - If user A has the notebook open and then user B swaps the permissions to view only user A can get stuck in a state where the local content is there but the API requests failed... We probs need an alert if the API response indicates no permissions and then some sort of refresh.